openedx/core/djangoapps/discussions/tests/test_views.py

```diff
@@ -387,6 +387,26 @@ def test_add_valid_configuration(self, provider_type):
         assert data['plugin_configuration'] == {'key': 'value'}
         assert data['lti_configuration'] == DEFAULT_LTI_CONFIGURATION
 
+    @ddt.data(
+        True,
+        False,
+    )
+    def test_enabled_configuration(self, enabled):
+        """
+        Test that enabling discussions unhides it.
```

Comment:
I'd clarify this comment too.

```suggestion
        Test that setting the "enabled" property for Discussions shows the Discussions tab.
```